tests/acceptance/test_examples.py

```diff
@@ -15,13 +15,15 @@
 
 
 @pytest.mark.parametrize("example_path", EXAMPLE_PATHS)
-def test___shipping_example___run___no_errors(example_path: Path):
-    local_system = nidaqmx.system.System.local()
+@pytest.mark.library_only
+def test___shipping_example___run___no_errors(example_path: Path, system):
+    # This test runs the example programs, which do not support gRPC.
+    # Hence this test is marked library_only.
```

Comment:
I was expecting something more like:
```
@pytest.mark.library_only  # The example programs do not support gRPC.
```

But now that I think about it, it would make sense for the `grpc_only` and `library_only` marks to have a `reason=` keyword argument like the skip/xfail marks. https://github.com/ni/nidaqmx-python/pull/387